src/nvim/nvim.cpp

```diff
@@ -564,10 +564,17 @@ bool NvimProcessKeyDown(Nvim *nvim, int virtual_key) {
 	return true;
 }
 
-void NvimOpenFile(Nvim *nvim, const char *file_name) {
+void NvimOpenFile(Nvim *nvim, const wchar_t *file_name) {
+
+	char utf8_encoded[MAX_PATH]{};
+	//if(!WideCharToMultiByte(CP_UTF8, 0, file_name, 1, 0, 0, NULL, NULL)) {
```

Comment:
Looks good to me, do you mind removing this commented section? Then I'll merge

Reply:
Yes. I just forgot about it.
The cleaned up code is in the commit below.